### ISSUE TITLE
[Remoto e Internacional] Vagas em React - diversas senioridades - Turing.com 

### ISSUE THREAD

**Issue by tatidirienzo** [react-brasil/vagas#2823]
Hey pessoal, tudo bem? Vim aqui trazer  em um formato um pouco diferente - uma listagem de vagas remotas e internacionais da Turing. 

Pra ser mais objetiva, classifiquei elas por senioridade e coloquei em destaque as principais stacks solicitadas por cada uma. Espero que seja útil :)

[Desenvolvedor Web Front-end](https://remotar.com.br/job/15898/turing.com/desenvolvedor-web-front-end-junior?utm_source=github)
Nível: Júnior e Pleno
Experiência com React e WordPress

[Desenvolvedor Full Stack com foco em Front-end](https://remotar.com.br/job/16163/turing.com/desenvolvedor-full-stack-com-foco-em-front-end?utm_source=github)
Nível: Pleno e Sênior
Experiência com React, React Native e JavaScript

[Desenvolvedor Front-end](https://remotar.com.br/job/16017/turing.com/desenvolvedor-front-end?utm_source=github)
Nível: Pleno e Sênior
Experiência com TypeScript, JavaScript e React.js

[Programador Front-end](https://remotar.com.br/job/16016/turing.com/programador-front-end?utm_source=github)
Nível: Pleno e Sênior
Experiência com React, Node.js e PostgreSQL

[Frontend Developer](https://remotar.com.br/job/15896/turing.com/frontend-developer?utm_source=github)
Nível Pleno e Sênior
Experiência com JavaScript, React and Vue.js

[Desenvolvedor Front-end React](https://remotar.com.br/job/15935/turing.com/desenvolvedor-front-end-react?utm_source=github)
Nível: Sênior
Experiencia com React, Redux e Typescript

Vale destacar que a Turing é uma empresa de desenvolvimento de software, então eles estão com muitas vagas abertas. O salário pode chegar à U$D 128.000 / ano, mas claro, vai depender do nível de conhecimento e posição. 

Por fim, destaco também que eu não faço parte do processo seletivo das vagas, apenas ajudo a Turing com a divulgação das mesmas através da plataforma da Remotar - a qual sou fundadora, ok?

## Como se candidatar

Ao clicar nos links acima você será redirecionado para a vaga publicada no site da Remotar, para avaliar o descritivo completo de cada uma. Caso tenha interesse, então basta clicar em "acessar" para ser redirecionado(a) para a respectiva vaga no site da Turing.

## Labels

- 🏢 Remoto
- 👦 Júnior
- 👨 Pleno
- 👴 Sênior
- ⚖️ PJ
- 💰 5k-10k
- 💰 10k-15k
- 💰 15k+

**Comment by davidbsoares** [comment]
O problema da Turing é que eles bloqueiam o perfil dos candidatos que não possuam exatos 3 anos de experiência (acho que é por causa disso que bloquearam o meu). Aí meu perfil tá todo completo lá só que não sou capaz de me candidatar a vagas.

**Comment by thejoaov** [comment]
> O problema da Turing é que eles bloqueiam o perfil dos candidatos que não possuam exatos 3 anos de experiência (acho que é por causa disso que bloquearam o meu). Aí meu perfil tá todo completo lá só que não sou capaz de me candidatar a vagas.

Eu tenho mais de 3 anos e mesmo assim, ainda estou bloqueado lá não sei por qual motivo.

**Comment by tatidirienzo** [comment]
> > O problema da Turing é que eles bloqueiam o perfil dos candidatos que não possuam exatos 3 anos de experiência (acho que é por causa disso que bloquearam o meu). Aí meu perfil tá todo completo lá só que não sou capaz de me candidatar a vagas.
> 
> Eu tenho mais de 3 anos e mesmo assim, ainda estou bloqueado lá não sei por qual motivo.

se quiser, me passa seu email que eu entro em contato com eles

**Comment by thejoaov** [comment]
> > > O problema da Turing é que eles bloqueiam o perfil dos candidatos que não possuam exatos 3 anos de experiência (acho que é por causa disso que bloquearam o meu). Aí meu perfil tá todo completo lá só que não sou capaz de me candidatar a vagas.
> > 
> > Eu tenho mais de 3 anos e mesmo assim, ainda estou bloqueado lá não sei por qual motivo.
> 
> se quiser, me passa seu email que eu entro em contato com eles

jvictorsantos852@gmail.com
Já fiz os testes e a interview com um Talent OPS, mas a aba de jobs continua bloqueada. Como citei antes, tenho quase 7 anos de exp.

**Comment by andrecampll** [comment]
@tatidirienzo a mesma coisa acontece comigo. Tenho 4 anos de experiência, já fiz todos os testes da minha stack de preferência, participei de interview com Talent OPS e mesmo assim ainda estou bloqueado.

**Comment by tatidirienzo** [comment]
@thejoaov em qual estágio você foi bloqueado, sabe dizer?

@andrecampll mesma coisa: pode me dizer em que estágio foi bloqueado e me mandar seu email também?

**Comment by andrecampll** [comment]
> > @thejoaov em qual estágio você foi bloqueado, sabe dizer?
> > @andrecampll mesma coisa: pode me dizer em que estágio foi bloqueado e me mandar seu email também?
> 
> Como eu tinha dito, fiz todos os testes, inclusive os da minha stack, já fiz a call com o talent ops. A aba de jobs continua bloqueada.

A mesma coisa para mim

**Comment by thejoaov** [comment]
> @thejoaov segue o retorno que tive:
> 
> _I checked this candidate in the DB and found that he has cleared the whole process and he should be matched with clients. Let me ask the tech team about this._
> 
> Aproveitando, ele questionou que a sua origem no banco de dados deles está como Linkedin, e não como este anúncio do Github. Você pode me confirmar só se chegou a fazer um acesso prévio via Linkedin?
> 
> @andrecampll aguardo seu email, ok?

Sim, foi por uma vaga que vi no linkedin. Mas como mencionei, estava bloqueado

**Comment by tatidirienzo** [comment]
@andrecampll e @thejoaov segue o retorno que tive:

_I talked to some internal teams and they are looking at the scenario at the moment. Currently, it's a speculation that these developers have cleared the process and have become a part of our developer pool. After this, the developers are matched with our clients as per client skills requirements, and then they start working on our clients' projects. So, they must be at the stage where they are in the developer pool but yet to be connected with clients, for which our team reaches out to developers when their candidature is shortlisted by the clients.
Anyhow, we are looking into this scenario more to get the exact picture. Meanwhile, can you please ask these developers to share some screenshots or screen recordings explaining the exact scenario where they feel stuck? That would be really helpful._ 

sendo assim, conseguem me enviar esses prints?

**Comment by thejoaov** [comment]
> @andrecampll e @thejoaov segue o retorno que tive:
> 
> _I talked to some internal teams and they are looking at the scenario at the moment. Currently, it's a speculation that these developers have cleared the process and have become a part of our developer pool. After this, the developers are matched with our clients as per client skills requirements, and then they start working on our clients' projects. So, they must be at the stage where they are in the developer pool but yet to be connected with clients, for which our team reaches out to developers when their candidature is shortlisted by the clients. Anyhow, we are looking into this scenario more to get the exact picture. Meanwhile, can you please ask these developers to share some screenshots or screen recordings explaining the exact scenario where they feel stuck? That would be really helpful._
> 
> sendo assim, conseguem me enviar esses prints?

<img width="1440" alt="Captura de Tela 2023-05-30 às 16 22 26" src="https://github.com/react-brasil/vagas/assets/45052080/72c2b357-9320-49c7-89db-6c81d0a4a419">

**Comment by andrecampll** [comment]
> @andrecampll e @thejoaov segue o retorno que tive:
> 
> _I talked to some internal teams and they are looking at the scenario at the moment. Currently, it's a speculation that these developers have cleared the process and have become a part of our developer pool. After this, the developers are matched with our clients as per client skills requirements, and then they start working on our clients' projects. So, they must be at the stage where they are in the developer pool but yet to be connected with clients, for which our team reaches out to developers when their candidature is shortlisted by the clients. Anyhow, we are looking into this scenario more to get the exact picture. Meanwhile, can you please ask these developers to share some screenshots or screen recordings explaining the exact scenario where they feel stuck? That would be really helpful._
> 
> sendo assim, conseguem me enviar esses prints?

![Screen Shot 2023-05-30 at 18 45 37](https://github.com/react-brasil/vagas/assets/53975579/5138bb2f-ffbe-4737-9719-60674ba64a4e)
